CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 * Updated Pebble Notices `get_notices` parameter name to `users=all` (previously `select=all`).
 * Added `Model.get_cloud_spec` which uses the `credential-get` hook tool to get details of the cloud where the model is deployed.
+* Updated code examples in the docstring of `ops.testing` from unittest to pytest style.
```

Comment:
I'm not sure whether we should include this in CHANGES.md or not - I've started a discussion in Matrix to figure out what our pattern should be :)

Reply:
I will resolve this for now and let's continue the discussion in the Matrix and we have another PR for tweaking CHANGES.md.

Reply:
Feel free to leave it in and whoever does the release next week can tidy it up, or pull it out and leave it for them to do - the first release with the new process will be a bit mixed anyway.